scripts/test_images.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -euo pipefail
+IFS=$'\n\t'
+
+source ./scripts/test_lib.sh
+
+function startContainer {
+    # run docker in the background
+    docker run -d --rm --name "${RUN_NAME}" "${IMAGE}"
+
+    # wait for etcd daemon to bootstrap
+    sleep 5
+}
+
+function runVersionCheck {
+    Out=$(docker run --rm "${IMAGE}" "${@}")
+    foundVersion=$(echo "$Out" | head -1 | rev  | cut -d" "  -f 1 | rev )
+    if [[ "${foundVersion}" != "${VERSION}" ]]; then
+        echo "error: Invalid Version. Got $foundVersion, expected $VERSION. Error: $Out"
+        exit 1
+    fi
+}
+
+# Can't proceed without docker
+if ! command -v docker >/dev/null; then
+    log_error "cannot find docker"
+    exit 1
+fi
+
+# You can't run darwin binaries in linux containers
+if [[ $(go env GOOS) == "darwin" ]]; then
+    echo "Please use linux machine for release builds."
+    exit 1
+fi
+
+# Pick defaults based on release workflow
+ARCH=$(go env GOARCH)
+REPOSITARY=${REPOSITARY:-"gcr.io/etcd-development/etcd"}
+if [ -n "$VERSION" ]; then
+    # Expected Format: v3.6.99-amd64
+    TAG=v"${VERSION}"-"${ARCH}"
```

Comment:
There is no suffix "-amd64" for amd64 arch in 3.4. 
```suggestion
    if [ "${ARCH}" != "amd64" ]; then
        TAG=v"${VERSION}"-"${ARCH}"
    else
        TAG=v"${VERSION}"
    fi
```

Reply:
Just pushed a version that removes the suffix, this one is better.